### PR TITLE
[refactor](stats) Refactor TableStatsMeta

### PR DESCRIPTION
**Pull request by Kikyou1997**
## Proposed changes

1. Add a abstraction for column stats status which is required so furthur optimization and feature development
2. Enable analyze test in p0 that disabled unexpectedly before

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisManager.java [hunk 1]
```diff
@@ -1069,10 +1076,18 @@ protected SimpleQueue<AnalysisInfo> createSimpleQueue(Collection<AnalysisInfo> c
...
     // we could make sure it would be analyzed again soon if user or system submit job for that column again.
     public void removeColStatsStatus(long tblId, String colName) {
-        TableStats tableStats = findTableStatsStatus(tblId);
+        TableStatsMeta tableStats = findTableStatsStatus(tblId);
         if (tableStats != null) {
             tableStats.removeColumn(colName);
         }
     }
 
+    public ColStatsMeta findColStatsMeta(long tblId, String colName) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisManager.java:1085]: not used?

Comment (Kikyou1997) [reply fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisManager.java:1085]: Need by display column stats

**Comment by Kikyou1997** [comment]
run buildall

**Review by englefly — DISMISSED** [review]

**Comment by Kikyou1997** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**